CONTRIBUTING.md

```diff
@@ -216,17 +216,17 @@ $ git commit -s -m 'This is my commit message'
 
 * In your FIBO installation directory, there is a directory called ".git", and a folder in there called "hooks".  This folder should have been created for you when you cloned the repository; if not, you can initialize your repository from the Git command line by changing directory to your fibo installation directory and typing: git init
 * The Git command line can be run from within the SourceTree tool by selecting "Open in terminal" from the Actions menu. Alternatively, you may set up a Git shell environment – see separate page 'Setting up the Git Shell' for details.
 * Running the initialization command should ensure that your .git environment exists and that it includes the /hooks directory. This does not automatically install or update the two files that you need to have in place for the Serializer to work. These are: 
-    * pre-commit (no file extension)
-    * rdf-toolkit.jar
-* These files are updated from time to time so it is recommended that you re-download these directly from the links below before proceeding.
-* Copy the file pre-commit to your hooks directory (use right-click on the link and select the save option).  Make sure that the name of the file is just pre-commit; your browser might want to append a suffix like .txt to it. 
+    * [pre-commit](etc/serialization/pre-commit) (no file extension)
+    * [rdf-toolkit.jar](etc/serialization/rdf-toolkit.jar)
+* These files are updated from time to time so it is recommended that you re-download these files before proceeding.
+* Copy the file [pre-commit](etc/serialization/pre-commit) to your hooks directory (use right-click on the link and select the save option).  Make sure that the name of the file is just pre-commit; your browser might want to append a suffix like .txt to it. 
 * Edit this pre-commit file to point to your Java JRE/JDK. To do this, open the pre-commit file in a text editor and search for "JAVA\_HOME". Edit the directory after JAVA\_HOME= to point to the location of your Java 8 (a.k.a version 1.8) or higher JRE/JDK, and save it. 
-* Copy the file rdf-toolkit.jar to your hooks directory by clicking on the link, viewing where the download was saved, and moving or copying it to the .git/hooks directory. 
+* Copy the file [rdf-toolkit.jar](etc/serialization/rdf-toolkit.jar) to your hooks directory by clicking on the link, viewing where the download was saved, and moving or copying it to the .git/hooks directory. 
```

Comment:
AFAIK the jar file does not work with "higher" than Java 8.
a) there's a stupid test in pre-commit that checks the major part of the version is "1"
b) the jar file makes use of unsafe Java features that are deprecated at Java 11.

Reply:
If the jar has a limitation like that, then we need to remove it.  I know that @abcoates is no longer in a position to do work directly for the council, but perhaps he could give us some advice about how to remove that dependency.